p2pfl/examples/mnist.py

```diff
@@ -169,6 +172,8 @@ def mnist(
             protocol=InMemoryCommunicationProtocol if use_local_protocol else GrpcCommunicationProtocol,  # type: ignore
             address=address,
             simulation=True,
+            aggregator=ScaffoldAggregator,
+            global_lr = 0.001,
```

Comment:
I agree with you that the aggregator should be instantiated by the user, and same for the rest of the Node parameters. It would be much more flexible. What do you think @pguijas ?